src/scenario.py

```diff
@@ -640,3 +642,9 @@ def run(self, strategy_name, options):
                 plt.setp(ax.get_xticklabels(), rotation=30, ha='right')
 
             plt.show()
+
+        ###########################
+        # get some numbers for testing
+        max_total_load = max(totalLoad) if totalLoad else 0
+
+        return(max_total_load)
```

Comment:
I would avoid giving that function a return value. Proposed change: make it a class attribute `self.max_total_load = ...`
In addition, totalLoad is always set (initialized as empty array), so the check is not needed.

Reply:
The check is in case the totalLoad is an empty list, in this case we cannot get the max of it. 

We could also add it as a class attrribute. But I think we will need to add some more parameters in order to test various scenarios/strategies, so quite alot more class attributes, that we only need for testing. Do you still like this better?

Reply:
You're right, I was thinking about `sum()`. For `max()`, this makes sense.
Regarding the return value: for many testing parameters, I would still prefer to save them as class attributes, even more so. Maybe they could be clustered into a testing or meta or whatever attribute.